PDK_Abstraction/FinFET14nm_Mock_PDK/FinFET_Mock_PDK_Abstraction_v1.json

```diff
@@ -0,0 +1,436 @@
+"VencA_L": [20],
+"VencA_H": [20],
+"VencP_L": [0],
+"VencP_H": [0],
+"MinNo": [1]
+},
+
+{
+"Layer": "M7",
+"LayerNo": 40,
+"Direction": "V",
+"Color": 1,
+"Width": [64],
+"Space": [80],
+"MinL": [360],
+"MaxL": ["NA"],
+"End-to-End": [70],
+"Offset": [0]
+},
+
+{
+"Layer": "V7",
+"LayerNo": 43,
+"Stack": "M7-M8",
+"WidthX": [64],
+"WidthY": [64],
+"SpaceX": [76],
+"SpaceY": [76],
+"VencA_L": [20],
+"VencA_H": [20],
+"VencP_L": [0],
+"VencP_H": [0],
+"MinNo": [1]
+},
+
+{
+"Layer": "M8",
+"LayerNo": 45,
+"Direction": "H",
+"Color": 1,
+"Width": [64],
+"Space": [80],
+"MinL": [360],
+"MaxL": ["NA"],
+"End-to-End": [70],
+"Offset": [0]
+},
+
+{
+"Layer": "V8",
+"LayerNo": 48,
+"Stack": "M8-M9",
+"WidthX": [64],
+"WidthY": [64],
+"SpaceX": [76],
+"SpaceY": [76],
+"VencA_L": [20],
+"VencA_H": [20],
+"VencP_L": [0],
+"VencP_H": [32],
+"MinNo": [1]
+},
+
+{
+"Layer": "M9",
+"LayerNo": 50,
+"Direction": "V",
+"Color": 1,
+"Width": [128],
+"Space": [128],
+"MinL": [200],
+"MaxL": ["NA"],
+"End-to-End": [70],
+"Offset": [0]
+},
+
+{
+"Layer": "V9",
+"LayerNo": 55,
+"Stack": "M9-M10",
+"WidthX": [128],
+"WidthY": [128],
+"SpaceX": [200],
+"SpaceY": [200],
+"VencA_L": [40],
+"VencA_H": [40],
+"VencP_L": [0],
+"VencP_H": [0],
+"MinNo": [1]
+},
+
+{
+"Layer": "M10",
+"LayerNo": 58,
+"Direction": "H",
+"Color": 1,
+"Width": [128],
+"Space": [128],
+"MinL": [500],
+"MaxL": ["NA"],
+"End-to-End": [128],
+"Offset": [0]
+},
+
+{
+"Layer": "V10",
+"LayerNo": 60,
+"Stack": "M10-M11",
+"WidthX": [128],
+"WidthY": [128],
+"SpaceX": [200],
+"SpaceY": [200],
+"VencA_L": [40],
+"VencA_H": [40],
+"VencP_L": [0],
+"VencP_H": [64],
+"MinNo": [1]
+},
+
+{
+"Layer": "M11",
+"LayerNo": 62,
+"Direction": "V",
+"Color": 1,
+"Width": [256],
+"Space": [256],
+"MinL": [260],
+"MaxL": ["NA"],
+"End-to-End": [128],
+"Offset": [0]
+},
+
+{
+"Layer": "V11",
+"LayerNo": 65,
+"Stack": "M11-M12",
+"WidthX": [256],
+"WidthY": [256],
+"SpaceX": [360],
+"SpaceY": [360],
+"VencA_L": [60],
+"VencA_H": [60],
+"VencP_L": [0],
+"VencP_H": [0],
+"MinNo": [1]
+},
+
+{
+"Layer": "M12",
+"LayerNo": 68,
+"Direction": "H",
+"Color": 1,
+"Width": [256],
+"Space": [256],
+"MinL": [1200],
+"MaxL": ["NA"],
+"End-to-End": [280],
+"Offset": [0]
+},
+
+{
+"Layer": "V12",
+"LayerNo": 70,
+"Stack": "M12-M13",
+"WidthX": [256],
+"WidthY": [256],
+"SpaceX": [360],
+"SpaceY": [360],
+"VencA_L": [60],
+"VencA_H": [60],
+"VencP_L": [0],
+"VencP_H": [472],
+"MinNo": [1]
+},
+
+{
+"Layer": "M13",
+"LayerNo": 72,
+"Direction": "V",
+"Color": 1,
+"Width": [1200],
+"Space": [1200],
+"MinL": [400],
+"MaxL": ["NA"],
+"End-to-End": [280],
+"Offset": [0]
+},
+
+{
+"Layer": "V13",
+"LayerNo": 75,
+"Stack": "M13-M14",
+"WidthX": [1200],
+"WidthY": [1200],
+"SpaceX": [1200],
+"SpaceY": [1200],
+"VencA_L": [100],
+"VencA_H": [100],
+"VencP_L": [0],
+"VencP_H": [0],
+"MinNo": [1]
+},
+
+{
+"Layer": "M14",
+"LayerNo": 78,
+"Direction": "H",
+"Color": 1,
+"Width": [1200, 1600, 2000, 2500, 4000, 5000, 10000, 20000],
```

Comment:
Adding a comment here purely for discussion. Is it better to have multiple width Spacing MinL / MaxL definitions within each layer or simply create multiple layers with shared layer numbers? I am not sure which will be easier to refer to in downstream passes.

Reply:
@stevenmburns Could use your input here as well.

Reply:
In my views, It is easier to handle a single layer with multiple Width / Space / MinL / MaxL options. If this is an issue we can discuss it in tomorrow's meeting

Reply:
Case in point: What happens with via layers connecting metal layers with differing widths? I would expect the via MinNo. would change at the very least. Not sure if enclosure changes as well.

Reply:
For some technologies, it might be a case. For that, we can have the via MinNo. list as you suggested.

Reply:
So one MinNo entry per pair of widths? Could work but may be clunky. Anyway. Resolving this conversation as of now. We can always revisit when we see a concrete example.